src/pages/Home.tsx

```diff
@@ -0,0 +1,11 @@
+import React from "react";
+import './style.css';
+
+export const Home = () => {
```

Comment:
type annotation missing

Reply:
Also use implicit return with arrow function if there's no other code in the function

```tsx
export const Home: React.FC = () => (
        <div className="centered">
            <h1>My Shopping App</h1>
            <p>Shop for all your favorite products in one place</p>
        </div>
);